### DISCUSSION TITLE
Stochastics - Steady to Transient

### DISCUSSION THREAD

**Discussion by MatiasAllay** [idaholab/moose#18184] [Q&A Modules: General] [answered]
Dear MOOSE,
My simulation consists of two scripts. First a script that runs steady state and second a transient state script that uses the output of the steady simulation as ICs. I was wondering of there is a way to run one script after the other one with the stochastic module to perform a sensitivity analysis. 
Thanks in advance!
Matias

**Comment by aeslaughter** [comment, marked as answer]
This should be possible. The main input file will need to use the `Transient` executioner with two time steps. At the first timestep you will run the steady solves, using a full solve multiapp that runs the input file for the steady. The second step will run a different full solve multiapp that runs the transient input file.

You will need to use the [TimePeriod](https://mooseframework.inl.gov/source/controls/TimePeriod.html) object in the main input to dictate with multiapp will run.

I would start by getting the main input file to run the steady solve, when you get that figured out work on adding the second part.

**Reply by MatiasAllay** [reply]
Thanks @aeslaughter I will let you know how it goes.
